ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/RequestHelper.java

```diff
@@ -153,4 +169,40 @@ public static String generateParameterValidationBlock(final Request request) {
 
         return builder.toString();
     }
+
+    public String getRequestObjectListType(final String requestName) {
+        switch(requestName) {
+            case "get_bulk_job_spectra_s3_request":
+                return "BULK_GET";
+
+            case "put_bulk_job_spectra_s3_request":
+                return "BULK_PUT";
+
+            case "verify_bulk_job_spectra_s3_request":
+            case "eject_storage_domain_spectra_s3_request":
+            case "eject_storage_domain_blobs_spectra_s3_request":
+            case "get_physical_placement_for_objects_spectra_s3_request":
+            case "get_physical_placement_for_objects_with_full_details_spectra_s3_request":
+            case "verify_physical_placement_for_objects_spectra_s3_request":
+            case "verify_physical_placement_for_objects_with_full_details_spectra_s3_request":
+                return "GET_PHYSICAL_PLACEMENT";
+
+            case "complete_multi_part_upload_request":
+                return "COMPLETE_MPU";
+
+            case "put_multi_part_upload_part_request":
+            case "put_object_request":
+                return "DATA";
+
+            case "delete_objects_request":
+                return "STRING_LIST";
+
+            case "get_blob_persistence_spectra_s3_request":
+            case "replicate_put_job_spectra_s3_request":
+                return "STRING";
+
+            default:
+                return null;
```

Comment:
Its an error state.  The templates will blow up if I don't have everything matched up correctly